controller/engine_controller.go

```diff
@@ -621,54 +595,29 @@ func (ec *EngineController) stopMonitoring(e *longhorn.Engine) {
 		return
 	}
 	stopCh <- struct{}{}
```

Comment:
You don't need to write into the channel you can just close it. So you can make the channel an unbuffered channel.
It's valid to read from a closed channel and it will return the default value for the channel continuously & immediately.

Reply:
Good suggestion. I will update it.